doc/integrating_applications/topics/adding_dropbox_connection_middle.adoc

```diff
@@ -0,0 +1,36 @@
+[id='adding-dropbox-connection-middle']
+= Adding a Dropbox connection as a middle connection
+
+In an integration, to add a Dropbox connection between the start and 
+finish connections:
+
+. Add the start and finish connections to the integration.
+. In the left panel, hover over the plus sign that is in the location
+where you want to add a Dropbox connection.
+. In the popup, click *Add a Connection*.
+. On the *Choose a Connection* page, click the Dropbox connection that you 
+want the integration to use. 
+
+. On the *Choose an Action* page, select one of the following actions:
++
+* *Upload a file to Dropbox* to add the current integration data to the
+Dropbox account that this connection accesses. To configure this
+action:
+.. In the *Remote Path* field, enter the path of the Dropbox folder 
```

Comment:
Here you'll need to specify the the file name too.

Reply:
Is this correct: 
In the Remote Path field, to upload multiple files, enter the path of the Dropbox folder in which to store the integration data. To upload one file, enter the Dropbox path and file name.

Reply:
The remote path can be a folder or a file. But the folder must be in the local File system, since we use the upload action only as an end or middle connection, you won't have a folder coming from another connection, so yes, this should be used to upload a single file.

Reply:
Is this correct:
In the Remote path field, to upload multiple files, enter the local path for the folder that you want to upload. Dropbox uses the same path to store the files. To upload one file, specify the local path and file name of the file you want to upload. Dropbox stores the file with the same path and name. 

Sorry if I am still getting this wrong. I understand that "Remote path" is the name of the Camel Dropbox option, but I am seeing two different explanations for it. One says it is the Dropbox destination and the other says that it is the original (local I guess) path.

Reply:
This is correct.

Reply:
Happy dance! Thanks!